### DISCUSSION TITLE
Toolbar background and item colors

### DISCUSSION THREAD

**Discussion by chrisgurney** [chrisgurney/obsidian-note-toolbar#28] [Ideas]
From this [discussion thread](https://www.reddit.com/r/ObsidianMD/comments/1bzwk24/comment/kyt288j/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) from [DMNK392](https://www.reddit.com/user/DMNK392/) on Reddit:

> What would be cool would be some custom design options like background color and buttons/links color.

**Comment by FelipeRearden** [comment]
> From this [discussion thread](https://www.reddit.com/r/ObsidianMD/comments/1bzwk24/comment/kyt288j/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) from [DMNK392](https://www.reddit.com/user/DMNK392/) on Reddit:
> 
> > What would be cool would be some custom design options like background color and buttons/links color.

Just to let you know that there is a plugin called **Style Settings** that might help you to achieve this 

https://github.com/mgmeyers/obsidian-style-settings

From README.md
> This plugin allows snippet, theme, and plugin CSS files to define a set of configuration options. It then allows users to see all the tweakable settings in one settings pane. Style Settings allows both toggling classes on and off the body element, as well as setting numeric, string, and color CSS variables.

I hope it helps 🙏

**Reply by chrisgurney** [reply]
Thank you for suggesting this @FelipeRearden ! 

I do like the idea of leveraging another popular plugin for customizations like this, versus having to build in settings for all sorts of styling options. CSS snippets are part of the solution for today, but this feels like a workable solution for tomorrow.

**Reply by FelipeRearden** [reply]
Okay! Glad to help @chrisgurney !!!!

**Reply by chrisgurney** [reply]
@FelipeRearden First commit! You have no idea how much time this has saved me: https://github.com/chrisgurney/obsidian-note-toolbar/commit/be2ae312d0a634d722616a1e45e760db89e5fd60

**Reply by FelipeRearden** [reply]
Thanks for the feedback @chrisgurney !!!!!
Based on past experiences I was almost sure that Style Settings could be a big time saver ⏱️

I wish you a fantastic day ☀️

**Comment by MrComputron** [comment]
On the idea of theming buttons (and just posted in discussion) would it be possible to have a greater list of icons, or allow for the use of any keyboard character or emoji?

**Reply by chrisgurney** [reply]
You can put emojis in the title of an item, and it will work. Just leave the icon blank (not set).

Would this work for you?

**Comment by chrisgurney** [comment]
The Style Settings plugin is now supported [in the beta](https://github.com/chrisgurney/obsidian-note-toolbar/discussions/40)!

Styles available include:

- Toolbar - Border color, Background color, Left + Right padding
- Toolbar Items - Text color, Text color (on hover), Background color (on hover)
- Toolbar Items (“button” style) - Background color

Read more about it here: https://github.com/chrisgurney/obsidian-note-toolbar/discussions/40

**Comment by chrisgurney** [comment]
Styling via Style Settings has now been [released](https://github.com/chrisgurney/obsidian-note-toolbar/discussions/40).

Emojis can be used in the title of any item; just leave the icon blank (not set).
